media-video/aegisub/aegisub-9999.ebuild

```diff
@@ -94,12 +92,14 @@ src_configure() {
 		$(use_with pulseaudio libpulse)
 		$(use_with spell hunspell)
 		$(use_with uchardet)
+		--disable-compiler-flags
 	)
 	econf "${myeconfargs[@]}"
 }
 
 src_compile() {
-	emake
+	# Concurrent builds seem to break the build process.
+	emake -j1
```

Comment:
I've constrained the build to one job, because I've had some hard to pin down build failures, and I suspect it's due to parallel building. I'd be grateful if someone with better autotools knowledge could debug this further.